### ISSUE TITLE
Tons of errors when synchronising 

### ISSUE THREAD

**Issue by Binero** [fosskers/aura#285]
When just performing a simple `sudo aura -Ayu` aura generates a lot of errors: 

```
10/25 22:59:33 [ERROR] CUID#11 - Download aborted. URI=http://ftp.hawo.stw.uni-erlangen.de/archlinux/community/os/x86_64/community.db.sig
Exception: [AbstractCommand.cc:350] errorCode=3 URI=http://ftp.hawo.stw.uni-erlangen.de/archlinux/community/os/x86_64/community.db.sig
  -> [HttpSkipResponseCommand.cc:215] errorCode=3 Resource not found
```

```
10/25 22:59:33 [NOTICE] GID#543baf44728478a5 - Download has already completed: /var/lib/pacman/sync/multilib.db

10/25 22:59:33 [NOTICE] Download complete: /var/lib/pacman/sync/multilib.db
```

It basically does that at least another 200 times, with different URLs. Eventually it'll say there are no packages requiring updates, while I know there are. 

Full log: http://hastebin.com/usaxucecel (takes some time to load, over a thousand lines! )

**Comment by rulatir** [comment]
@Binero if your definition of "working like charm" is "not actually doing any download acceleration whatsoever"...

**Comment by tblancher** [comment]
https://bbs.archlinux.org/viewtopic.php?pid=1254940#p1254940

**Comment by mhicklen** [comment]
In case anyone else finds this helpful - I was encountering endless errors just like this post (this was one of the first things I found via a web search) and here's my take. When you modify SigLevel= in the main section of pacman.conf, it applies it to all repositories.

To fix this, I just added '''SigLevel = PackageRequired''' to all the repos I had enabled - the forum post linked above only mentioned [core] but I had to also add it to [extra] which I had enabled, as well as [community] which I also had enabled.

Hope this helps someone.
